### DISCUSSION TITLE
How to Fix the Botframework Endpoint?

### DISCUSSION THREAD

**Discussion by narita-takeru** [OfficeDev/microsoft-365-agents-toolkit#12177] [Q&A]
When deploying the app from the Teams Toolkit in VSCode, the endpoint changes every time.

<img width="200" alt="スクリーンショット 2024-08-07 13 36 31" src="https://github.com/user-attachments/assets/1d8a3c0a-b4b9-45e4-9c8b-90f1113d42d2">

env/.env.local
```
BOT_ENDPOINT=https://xxxxx.asse.devtunnels.ms
BOT_DOMAIN=xxxxx.asse.devtunnels.ms
```

This becomes a hindrance if there is additional development after the app has been published. Are there any best practices or documentation on how to handle this operationally?

**Comment by therealjohn** [comment]
That's intended during development to avoid timeout issues with the tunnels. What's the use case where you need that to be long-lived for publishing?
